src/domain/models/east_mediterranian_mal/Malaria.jsx

```diff
@@ -0,0 +1,116 @@
+import _ from "lodash";
+import i18n from "../../../locales";
+
+const MalariaRepoDescription = () => {
+    return (
+        <>
+            <p>{i18n.t("Malaria endemic countries in the region are invited to upload a subset of their malaria related data (by district and month) into this DHIS2 instance.")}</p>
+            <p>{i18n.t("Some countries are already implementing DHIS2 for their routine HMIS and have already adopted the malaria modules. These countries can ready link the two systems. Others are using other HMIS systems and will need to use the Bulk Load app to load the data.")}</p>
+            <p style={{ textAlign: "left" }}>{i18n.t("This repository will be useful for the following:")}</p>
```

Comment:
Move to styles object or styled-components or useStyles, whatever the rest of the app is using.